txcache/trackedBlock.go

```diff
@@ -34,40 +45,46 @@ func (tb *trackedBlock) compileBreadcrumbs(txs []*WrappedTransaction) {
 func (tb *trackedBlock) compileBreadcrumb(tx *WrappedTransaction) {
 	sender := tx.Tx.GetSndAddr()
 	feePayer := tx.FeePayer
-	nonce := tx.Tx.GetNonce()
+	initialNonce := tx.Tx.GetNonce()
+	latestNonce := tx.Tx.GetNonce()
 
 	// compile for sender
-	senderBreadcrumb := tb.getBreadcrumb(string(sender), nonce)
+	senderBreadcrumb := tb.getOrCreateBreadcrumb(string(sender), core.OptionalUint64{
```

Comment:
Maybe have two such functions:
 - `getOrCreateBreadcrumb()`
 - `getOrCreateBreadcrumbWithNonce()`

And call the one without nonce for the fee payer flow?